src/dasherize.ts

```diff
@@ -1,6 +1,14 @@
 export const dasherize = (str: unknown): string =>
-  String(str)
+  String(typeof str === 'symbol' ? str.description : str)
     .replace(/([A-Z]($|[a-z]))/g, '-$1')
     .replace(/--/g, '-')
     .replace(/^-|-$/, '')
     .toLowerCase()
+
+export const mustDasherize = (str: unknown, type = 'property'): string => {
+  const dashed = dasherize(str)
+  if (!dashed.includes('-')) {
+    throw new DOMException(`${type}: ${String(str)} is not a valid ${type} name`, 'SyntaxError')
+  }
+  return dashed
+}
```

Comment:
This isn't used any where? Should this be in some other PR?

Reply:
Oh this was a rebase error. :shrug: doesn't do any harm staying here.